CMakeLists.txt

```diff
@@ -31,6 +31,17 @@ endif("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
 # Set default flags (from opus-ng build)
 set(CMAKE_C_FLAGS "-O2 -fvisibility=hidden -fstack-protector-strong -W -Wall -Wextra -Wcast-align -Wnested-externs -Wshadow -Wstrict-prototypes")
 
+# Python tells us the CFLAGS we need for Embedding Python in a C lib
+# The default PYTHON_CONFIG works on Ubuntu 22 but will likely need to be changed for other distros
+if(NOT PYTHON_CONFIG)
```

Comment:
Why not use `FindPython()` to get this info? More info: https://cmake.org/cmake/help/latest/module/FindPython.html

Reply:
Additionally, it seems to also support finding where numpy's located (since the include path can apparently be different from the main Python one).

Reply:
Hi @tmiw - if you want to try that out, pls feel free.  I'm kinda focused on getting to a C library you can link with :slightly_smiling_face:

I'm also stuck on the GitHub workflow breaking if you would like to take a look at that? Otherwise I'll disable it for now and come back to it later.  The ctests work OK for me locally.

Reply:
The workflow issue looks like it was due to a compiler error that I was also getting locally:

```
[ 75%] Building C object src/CMakeFiles/radae_tx.dir/rade_api.c.o
/Users/mooneer/devel/radae/src/rade_api.c:234:29: warning: unused parameter 'model_file' [-Wunused-parameter]
  234 | struct rade *rade_open(char model_file[]) {
      |                             ^
/Users/mooneer/devel/radae/src/rade_api.c:285:10: error: use of undeclared identifier 'tx_out_eoo'
  285 |   assert(tx_out_eoo != NULL);
      |          ^
1 warning and 1 error generated.
make[2]: *** [src/CMakeFiles/radae_tx.dir/rade_api.c.o] Error 1
make[1]: *** [src/CMakeFiles/radae_tx.dir/all] Error 2
make: *** [all] Error 2
```

Fixing that appears to have fixed the build on GitHub.

BTW `FindPython` does appear to also work on GitHub as well as on macOS (using my local venv). Still need to try Windows but I don't *think* there'll be any problems if I can get the `conda` environment I have onto another system.

Reply:
It's now building and ctests running fine on an Ubuntu 20 machine too :slightly_smiling_face: